credit-risk/src/config.py

```diff
@@ -1,4 +1,4 @@
-from .predictor import Predictor
+from .predictor import ChurnPredictor
```

Comment:
What's the reason for calling this `ChurnPredictor` in this version? I believe this is a credit risk prediction, so not churn specific

Reply:
Good catch. Updated.